src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -178,13 +178,24 @@ export default class ResultsViewPage extends React.Component<
 
         this.resultsViewPageStore = initStore(props.appStore);
 
+        this.props.routing
+
         getBrowserWindow().resultsViewPageStore = this.resultsViewPageStore;
     }
 
     private handleTabChange(id: string, replace?: boolean) {
         this.props.routing.updateRoute({}, `results/${id}`, false, replace);
     }
 
+    @autobind
+    private handleSilentUrlUpdate(url:string) {
```

Comment:
this should become much easier when i finish my URLWrapper PR, which should allow features to observe separate parameters of the URL instead of having to observe the entire thing.

Reply:
@pvannierop is it possible to wait until @alisman 's PR is merged? @alisman how much longer will it be?

Reply:
>  Just to confirm, the point is to update sortMode without throwing the oncoprint into loading state, right?

Yes, that is correct.

> @pvannierop is it possible to wait until @alisman 's PR is merged?

For sure. No problem. I am happy to see there may be a solution.